exe/facter.cc

```diff
@@ -299,6 +281,9 @@ int main(int argc, char **argv)
             log(level::warning, "Facter was called recursively, skipping external facts. Add '--no-external-facts' to silence this warning");
           } else {
             environment::set("INSIDE_FACTER", "true");
+            if (vm.count("external-dir")) {
+                external_directories.push_back(vm["external-dir"].as<string>());
```

Comment:
I don't know, honestly that predates me. /cc @peterhuene 

You can use multiple arguments on the command-line via `multitoken`, but I don't think we actually want to. So this should maybe just be a string everywhere (that we `move` into our internal vectors).